transformations/weekday_month_abbreviation/transformation.py

```diff
@@ -0,0 +1,85 @@
+import re
+from typing import List
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+def weekday_month_abbreviate(text, max_outputs=1):
+    abbdreviations = {
```

Comment:
This should be hard coded. I would like to see this as a either a JSON or XML config file so that if for example multi-lingual support is added in the future it can be done so by extended your config file.

Reply:
Change `abbdreviations` to `abbreviations`.

Reply:
Thank you @Saad-Mahamood for the suggestion. I have hard coded the dictionaries in 2 JSON files in the commit ea5f5d59ad84b451ac882d387eea80c5bde6115e